.github/workflows/health_checks.yml

```diff
@@ -47,7 +47,7 @@ on:
         description: 'Node versions list (as JSON array).'
         required: false
         type: string
-        default: '["18", "20"]'
+        default: '["18", "22"]'
```

Comment:
whats support guidance for 20? since we don't pull 18 up to 20

Reply:
I left a note in PR description about this.

The 18 is minimum we support. The 22 then becomes the max we know we support. I think this is enough. The presence of 18 should assert that we can handle "old node".

We definitely don't want to run all e2e tests on 18,20 and 22.

A lightweight solution that we could do is to run install-build-unit-tests on 18, 20 and 22, but I'm still not convinced we need this.

@Amplifiyer @rtpascual wdyt?

Reply:
If most of the customers are on node 20, then I see some value of running at least some sanity tests with it. Though it's true that if the library functions well on `22` and on `18`, every version in between is covered, the real life situations situations such as special logic or considerations for `18` might break that.

My thought is to have some coverage for the majority use case. As soon as the `20` is no longer the majority shareholder, we can drop it.

Reply:
I agree. While testing on `22` should cover `20`, there may be edge cases we might not catch if we're not explicitly testing on `20` and this goes for whenever future node versions are released. Where we set the line for dropping testing a node version we need to decide at some point.

Reply:
Ok.
Would `install-build-unit-tests` be good enough or do we want to bring some e2e into the mix ?

Reply:
ok, look like majority is node `18` so we definitely don't need extensive coverage for `20`. Just the build and unit tests sounds good to me.

Reply:
Alright, we discussed this offline a bit.
Decided to try just adding Node 22 for now. We'll re-evaluate approach later. We still have room for ~164 more jobs.